README.md

```diff
@@ -4,3 +4,67 @@ RateLimit.js
 Status](https://travis-ci.org/dudleycarr/ratelimit.js.svg)](https://travis-ci.org/dudleycarr/ratelimit.js)
 
 A NodeJS library for rate limiting using sliding windows stored in Redis.
+
+### Express Middleware
+
+Construct rate limiter and middleware instances:
+
+```coffee
+RateLimitjs = require 'ratelimit.js'
```

Comment:
Or some other name given that multiple things are exported. the JS seems odd even though it's in the package name.